src/KKManager/ModpackTool/Data/ModpackToolConfiguration.cs

```diff
@@ -15,32 +15,44 @@ namespace KKManager.ModpackTool
+        public ValidatedString FailFolder { get; } = new ValidatedString(Directory.Exists);
+        public ValidatedString BackupFolder { get; } = new ValidatedString(Directory.Exists);
+        public ValidatedString LooseFilesFolder { get; } = new ValidatedString(Directory.Exists);
 
-        public ValidatedString Game1Short { get; } = new(s => s.All(x => char.IsUpper(x) || char.IsDigit(x)));
-        public ValidatedStringWrapper Game1Longs { get; } = new(s => Game1LongsList = ZipmodEntry.GameNamesStrToList(s), () => ZipmodEntry.GameNamesListToStr(Game1LongsList), ZipmodEntry.GameNamesVerifierLoose);
+        public ValidatedString Game1Short { get; } = new ValidatedString(s => s.All(x => char.IsUpper(x) || char.IsDigit(x)));
+        public ValidatedStringWrapper Game1Longs { get; } = new ValidatedStringWrapper(s => Game1LongsList = ZipmodEntry.GameNamesStrToList(s), () => ZipmodEntry.GameNamesListToStr(Game1LongsList), ZipmodEntry.GameNamesVerifierLoose);
         public static IReadOnlyCollection<string> Game1LongsList { get; private set; } = Array.Empty<string>();
 
-        public ValidatedString Game2Short { get; } = new(s => s.All(x => char.IsUpper(x) || char.IsDigit(x)));
-        public ValidatedStringWrapper Game2Longs { get; } = new(s => Game2LongsList = ZipmodEntry.GameNamesStrToList(s), () => ZipmodEntry.GameNamesListToStr(Game2LongsList), ZipmodEntry.GameNamesVerifierLoose);
+        public ValidatedString Game2Short { get; } = new ValidatedString(s => s.All(x => char.IsUpper(x) || char.IsDigit(x)));
+        public ValidatedStringWrapper Game2Longs { get; } = new ValidatedStringWrapper(s => Game2LongsList = ZipmodEntry.GameNamesStrToList(s), () => ZipmodEntry.GameNamesListToStr(Game2LongsList), ZipmodEntry.GameNamesVerifierLoose);
         public static IReadOnlyCollection<string> Game2LongsList { get; private set; } = Array.Empty<string>();
 
-        public ValidatedString Game3Short { get; } = new(s => s.All(x => char.IsUpper(x) || char.IsDigit(x)));
-        public ValidatedStringWrapper Game3Longs { get; } = new(s => Game3LongsList = ZipmodEntry.GameNamesStrToList(s), () => ZipmodEntry.GameNamesListToStr(Game3LongsList), ZipmodEntry.GameNamesVerifierLoose);
+        public ValidatedString Game3Short { get; } = new ValidatedString(s => s.All(x => char.IsUpper(x) || char.IsDigit(x)));
+        public ValidatedStringWrapper Game3Longs { get; } = new ValidatedStringWrapper(s => Game3LongsList = ZipmodEntry.GameNamesStrToList(s), () => ZipmodEntry.GameNamesListToStr(Game3LongsList), ZipmodEntry.GameNamesVerifierLoose);
         public static IReadOnlyCollection<string> Game3LongsList { get; private set; } = Array.Empty<string>();
 
-        public IEnumerable<string> AllAcceptableGameLongNames => Game1LongsList.Concat(Game2LongsList).Concat(Game3LongsList);
+        public static IEnumerable<string> AllAcceptableGameLongNames => Game1LongsList.Concat(Game2LongsList).Concat(Game3LongsList);
```

Comment:
This can be an issue in the future if different implementations are added, there's no point to change this to static.